test/Base.t.sol

```diff
@@ -218,6 +223,22 @@ abstract contract Base_Test is Assertions, Calculations, Constants, DeployOptimi
         });
     }
 
+    /*//////////////////////////////////////////////////////////////////////////
+                            CALL EXPECTS - MERKLE LOCKUP
```

Comment:
> But then we should also move `CALL EXPECTS - LOCKUP` to `Periphery.t.sol `?

yeah, sure, sounds good

> . My view is that Base contracts works too.

the problem i see with is that the contract becomes extra populate and might be hard to follow. but technically it works there, yes


> never understood what dictates what should go in `Periphery.t.sol` because core does not have such a contract

core does not have it, as what could be placed there, it is needed in the periphery as well. also, in core we have `Integration.t.sol`, which doesn't exist in periphery dir